openfasoc/generators/glayout/glayout/flow/blocks/elementary/FVF/fvf.py

```diff
@@ -0,0 +1,164 @@
+from glayout.flow.pdk.mappedpdk import MappedPDK
+from glayout.flow.pdk.sky130_mapped import sky130_mapped_pdk
+from gdsfactory.cell import cell
+from gdsfactory.component import Component
+from gdsfactory import Component
+from glayout.flow.primitives.fet import nmos, pmos, multiplier
+from glayout.flow.pdk.util.comp_utils import evaluate_bbox, prec_center, prec_ref_center, align_comp_to_port
+from glayout.flow.pdk.util.snap_to_grid import component_snap_to_grid
+from glayout.flow.pdk.util.port_utils import rename_ports_by_orientation
+from glayout.flow.routing.straight_route import straight_route
+from glayout.flow.routing.c_route import c_route
+from glayout.flow.routing.L_route import L_route
+from glayout.flow.primitives.guardring import tapring
+from glayout.flow.pdk.util.port_utils import add_ports_perimeter
+from glayout.flow.spice.netlist import Netlist
+from glayout.flow.primitives.via_gen import via_stack
+from gdsfactory.components import text_freetype, rectangle
+
+def fvf_netlist(fet_1: Component, fet_2: Component) -> Netlist:
+
+         netlist = Netlist(circuit_name='FLIPPED_VOLTAGE_FOLLOWER', nodes=['VIN', 'VBULK', 'VOUT', 'Ib'])
+         
+         netlist.connect_netlist(fet_1.info['netlist'], [('D', 'Ib'), ('G', 'VIN'), ('S', 'VOUT'), ('B', 'VBULK')])
+         netlist.connect_netlist(fet_2.info['netlist'], [('D', 'VOUT'), ('G', 'Ib'), ('S', 'VBULK'), ('B', 'VBULK')])
+
+         return netlist
+
+def sky130_add_fvf_labels(fvf_in: Component) -> Component:
```

Comment:
Can this function be made pdk agnostic?

Reply:
This function is only used for LVS check

Reply:
That is fine. Is it specific to the sky130 pdk? If so, can it be made generic using the `MappedPDK` class?

Reply:
It uses layers which are not mapped in Mappedpdk(as far as I know).

Reply:
Oh, in that case it is fine.

Reply:
Can it be  merged then ?:) I want to make PRs for some other stuffs too